src/searchPage/searchPage.html

```diff
@@ -1,9 +1,8 @@
 <swt-filter-panel></swt-filter-panel>
-<main class="container">
-    <div class="content-wrapper">
-        <swt-searchbar></swt-searchbar>        
-        <h4 class="search-results">Total results</h4>
-        <swt-search-results></swt-search-results>
-    </div>
-</main>
+<div class="content-wrapper">
```

Comment:
Content-wrapper is a very widely used class name. Might be a good idea to be more specific here

Reply:
The class was unnecessary, and margin from the class was added to header component.